src/D2L.Bmx/Aws/AwsCredsCache.cs

```diff
@@ -0,0 +1,65 @@
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+namespace D2L.Bmx.Aws;
+
+internal class AwsCredsCache() {
+	private static readonly JsonSerializerOptions _options =
+		new() { DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull };
+
+	public void SaveToFile( string Org, string User, AwsRole role, AwsCredentials credentials ) {
+		if( !Directory.Exists( BmxPaths.BMX_DIR ) ) {
+			return;
+		}
+		if( Org == "" || User == "" ) { //It's not set in config file
```

Comment:
You can use string.IsNullOrEmpty instead of `== ""` and we do that in other places already 

https://github.com/Brightspace/bmx/pull/384/files#diff-75585c62d6f9197bcc885d332bb0e7a1d207d5e2b9389a4fa04107c92d9289a6R23